mungegithub/mungers/matchers/comment/notification.go

```diff
@@ -33,7 +33,7 @@ type Notification struct {
 
 var (
 	// Matches a notification: [NOTIFNAME] Arguments
-	notificationRegex = regexp.MustCompile(`^\[([^\]\s]+)\]\s?(.*)$`)
+	notificationRegex = regexp.MustCompile(`(?m:^\[([^\]\s]+)\] *?(.*)$)`)
```

Comment:
- Do we need to capture this group: Yes, it's the name of the notification
- Why do we include \s twice: No, we are actually excluding \s (confusing with all those squared brackets ...)